docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,99 @@
+ solidity program do not pass the specification.
+It uses mutations from the [Gambit](https://github.com/Certora/gambit)
+  mutation generator.
+It also allows users to include manually generated mutants.
+If a mutated program passes the specification,
+it may indicate that the specification is vacuous or not rigorous enough.
+In the rest of the document,
+  we refer to both the mutation generator and the verifier as Gambit.
+
+## Installations and Setup
+
+You will require java to run the mutation testing jar.
+Installation instructions can be found [here](https://www.java.com/en/download/help/download_options.html).
+
+## Running the Mutation Verifier
+
+- Example:
+```
+java -ea -jar $CERTORA/certora_jars/MutationTest.jar /path/to/config/file/Example.conf
+```
+
+- **NOTE: if a path has spaces, put quotes around it to ensure correct parsing by Kotlin's libraries**
+
+- Gambit supports `--staging`!
+
+**However, Gambit currently has trouble with
+`--send_only` and `--cloud` in the run scripts.
+If you have these flags, please remove them for now.**
+Apologies for the temporary inconvenience!
+
+The tool expects a configuration file (extension `.conf` is required)
+which is a JSON object.
+Here is an example configuration file:
+
+```
+{
+    "project_folder" : "Test/10Power",
+    "run_script" : "Test/10Power/runDefault.sh",
+    "gambit" : {
+        "filename" : "Test/10Power/TenPower.sol",
+        "solc" : "solc8.10"
+    },
+    "staging" : "master",
+    "manual_mutations" : {
+        "path/to/original/file.sol" : "path/to/manual/mutations/",
+        ...
+    },
+    ...
+}
+```
+
+Required Keys for the JSON Configuration File:
+- `"project_directory"` : the directory containing the original CVT project on which to perform mutation testing
+- `"run_script"` : the bash script used to run verification on the original project, usually `project_directory/run.sh` or similar
+- `"gambit"` : the JSON configuration element for invoking Gambit. May be a path to a gambit configuration file
+or the explicit JSON element contained therein. Refer to the open source Gambit repository for more details regarding Gambit configuration files.
+
+Optional Keys for the JSON Configuration File:
+- `"num_threads"` : the maximum number of threads to use for verification, as an integer
+- `"manual_mutations"` : optionally supplement the random mutant generation with your own manually-written mutants.
+Expects a JSON object whose keys are the paths to the original files and whose values are paths to directories containing
+manually-written mutants as `.sol` files. **IMPORTANT:** any manual mutations files provided must follow the naming
+convention `OriginalFileName.<unique-name>.sol`, where `unique-name` is a string ID unique with respect to the other
+manual mutants. It is recommended to use `mN` for brevity, where `N` is a unique integer.
+- `"offline"` : run mutation testing without internet connection, skipping the UI output and other web functions.
```

Comment:
This should be included in the debugging section, since non-internal users can't run locally.